### PR TITLE
[ADD] display the status of the dispute in contract summary

### LINKED ISSUE #48: Display the status of the dispute in contract summary
Currently, only a number is displayed which is not informative.
We should display the status in plain English: 
``` js
enum Status {NoDispute, WaitingPartyA, WaitingPartyB, DisputeCreated, Resolved}
```

### PR DESCRIPTION
**Pull request by epiqueras**
- Create Enum folder for Enum definitions.

Closes #48

### CODE REVIEW

**Comment by Bernabe-Felix** [comment]
@epiqueras 
Wouldn't be better to use `STATUS_TO_STATE`? is the exact same thing and has already been used for this required behaviour

**Comment by epiqueras** [comment]
@Bernabe-Felix oops, missed that. I updated the PR.

**Comment by n1c01a5** [comment]
thanks @epiqueras for your contribution and @Bernabe-Felix for the review
